gradle/groovyProject.gradle

```diff
@@ -49,12 +65,15 @@ testTasks.all { task ->
             }
         }
     }
-    if (javaVersion.java7) {
+    executable = file("${javaInstallationForTest.javaHome}/bin/java")
+    environment['JAVA_HOME'] = javaInstallationForTest.javaHome.absolutePath
```

Comment:
Is this necessary? Doesn't setting the `executable` suffice?

Reply:
It should. I made this the same as what we do in [gradle/java9.gradle](https://github.com/gradle/gradle/blob/405f0a80666137cd659b09f93b163652dd779056/gradle/java9.gradle#L158).